packages/aws-cdk-lib/aws-codedeploy/lib/server/deployment-group.ts

```diff
@@ -262,6 +273,12 @@ export class ServerDeploymentGroup extends DeploymentGroupBase implements IServe
     this._autoScalingGroups = props.autoScalingGroups || [];
     this.installAgent = props.installAgent ?? true;
     this.codeDeployBucket = s3.Bucket.fromBucketName(this, 'Bucket', `aws-codedeploy-${cdk.Stack.of(this).region}`);
+    this.loadBalancers = props.loadBalancers || (props.loadBalancer ? [props.loadBalancer]: undefined);
+
+    if (this.loadBalancers && this.loadBalancers.length === 0) {
+      throw new Error('Cannot set empty loadBalancers array');
```

Comment:
```suggestion
      throw new Error('loadBalancers must be a non-empty array');
```
Formatting. Also, a unit test needs to be added to verify that the error is thrown if `loadBalancers: []`.